File: Backend/src/main/java/com/bachld/project/backend/dto/response/hoidong/HoiDongListItemResponse.java
package com.bachld.project.backend.dto.response.hoidong;

import lombok.*;
import lombok.experimental.FieldDefaults;
import java.time.LocalDate;
import java.util.List;

@Builder
@Data
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class HoiDongListItemResponse {
    Long id;
    String tenHoiDong;
    LocalDate thoiGianBatDau;
    LocalDate thoiGianKetThuc;
    String loaiHoiDong;
}

File: Android/app/src/main/java/com/bachld/android/data/dto/response/hoidong/HoiDongListItemResponse.kt
package com.bachld.android.data.dto.response.hoidong

data class HoiDongListItemResponse(
    val id: Long,
    val tenHoiDong: String,
    val thoiGianBatDau: String,
    val thoiGianKetThuc: String,
    val loaiHoiDong: String?
)